syntax = "proto3";

import "protos/recognize_ingredients.proto";
import "protos/select_ingredient.proto";

option csharp_namespace = "IntelliCook.IngredientRecognition.Client";

service IngredientRecognitionService {
    rpc RecognizeIngredientsStream (stream RecognizeIngredientsStreamRequest) returns (RecognizeIngredientsResponse);
    rpc SelectIngredientStream (stream SelectIngredientStreamRequest) returns (stream SelectIngredientStreamResponse);
}
